### PR TITLE
fix setting camera transformation

### PR DESCRIPTION
**Pull request by bucketzxm**
how  to reproduce: 
```viewer = env.GetViewer(); viewer.SetCamera(viewer.GetCameraTransform(), viewer.GetCameraDistanceToFocus())```

### CODE REVIEW

File: plugins/qtosgrave/qtosgviewer.cpp [hunk 1]
```diff
@@ -1441,7 +1441,7 @@ RaveTransform<float> QtOSGViewer::GetCameraTransform() const
     boost::mutex::scoped_lock lock(_mutexGUIFunctions);
```
Comment (braineo) [inline plugins/qtosgrave/qtosgviewer.cpp:1441]: Probably no need for this lock?

**Comment by braineo** [comment]
Actually I would remove `_Tcamera`, and let `void QtOSGViewer::_SetCameraTransform()` take a transform.

File: plugins/qtosgrave/qtosgviewer.cpp (hunk 1 above)
Comment (bucketzxm) [reply plugins/qtosgrave/qtosgviewer.cpp:1439]: other gui functions could change the camera transformation .. ?

**Review by braineo — APPROVED** [review]
LGTM

**Pull request merged**